std/experimental/allocator/mallocator.d

```diff
@@ -198,7 +212,8 @@ struct AlignedMallocator
     /**
     Forwards to $(D alignedAllocate(bytes, platformAlignment)).
     */
-    @nogc @trusted void[] allocate(size_t bytes) shared
+    @trusted @nogc nothrow
```

Comment:
Even if `alignedAllocate` cannot provide a memory safe interface, this function can be `@trusted` if `alignment`/`platformAlignment` always satisifes the conditions of `posix_memalign`.